campaigns/mofo-eoy-2016/snippet.html

```diff
@@ -0,0 +1,328 @@
+{#
+2016 fundraising snippet.
+
+Variables:
+
+  @text: main text - Text of snippet.
+  @icon: image - Snippet icon. 64x64px. SVG or PNG preferred.
+  @button_label: Required. Text for donate button.
+  @donation_form_url: Required. Url to the donation form.
+
+  @button_color: Default blue. Red and green also options.
+
+  @locale: Default en-US. String for the locale code.
+  @currency_code: Default usd. The code for the currency. Examle gbp, cad, usd.
+
+  @small_buttons: checkbock - Scales the buttons down.
+  @donation_amount_first: number. Required. The amount for the first button.
```

Comment:
Here you say it's required and later it looks it can be skipped? https://github.com/mozmar/snippets/pull/88/files#diff-90c0ff362679dfe399e92a5a6fc3182fR217